src-docs/src/views/expression/expression_example.js

```diff
@@ -114,5 +141,81 @@ export const ExpressionExample = {
       snippet: stringingSnippet,
       demo: <Stringing />,
     },
+    {
+      title: 'Columns layout',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: columnsSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: columnsHtml,
+        },
+      ],
+      text: (
+        <div>
+          <p>
+            There might be cases where displaying a number of{' '}
+            <strong>EuiExpression</strong>s in a paragraph is not ideal. For
+            example, when both the <EuiCode>description</EuiCode> and the{' '}
+            <EuiCode>value</EuiCode> are variable or when the values they can
+            take can be quite long. To use a columns layout instead, pass{' '}
+            <EuiCode>columns</EuiCode> to the <EuiCode>display</EuiCode> prop.
```

Comment:
I find it easier for the consumer to read the right way to pass the prop when it's displayed as `prop="value"`.

```suggestion
            <EuiCode language="ts">{'display="columns"'}</EuiCode>.
```